storage/innobase/os/os0file.cc

```diff
@@ -10689,6 +10723,12 @@ Encryption::decrypt_log_block(
 	/* Reset the encrypted flag. */
 	log_block_set_encrypt_bit(ptr, false);
 
+	/* recalculate the checksum */
```

Comment:
This comment is obvious
If there has to be a comment here, better to explain, why the checksum has to be recalculated at this point